RxExample/RxExample/Examples/GitHubSearchRepositories/GitHubSearchRepositoriesViewController.swift

```diff
@@ -50,8 +50,15 @@ class GitHubSearchRepositoriesViewController: ViewController, UITableViewDelegat
                     : Observable.empty()
             }
 
-        let searchResult = searchBar.rx_text.asDriver()
+        let searchText = searchBar.rx_text.asDriver()
             .throttle(0.3)
+        let searchTapped = searchBar.rx_searchTapped.asDriver()
+            .flatMap { $0.text.flatMap { Driver.just($0) } ?? Driver.empty() }
+        let cancelTapped = searchBar.rx_cancelTapped.asDriver()
+            .map { _ in "" }
+      
+        let searchResult = Driver.of(searchText, searchTapped, cancelTapped)
```

Comment:
Take a look at `withLatestFrom` operator and how we use it to get an idea how to write this in more idiomatic and simpler way :)